### PR TITLE
#161178533 Fix Time to read for articles response

### PR DESCRIPTION
**Pull request by CEOehis**
#### What does this PR do?
Implement fix to return `timeToRead` with article response

#### Any background context you want to provide?
The previous implementation excluded this field.

#### What are the relevant pivotal tracker stories?
[#161178533](https://www.pivotaltracker.com/story/show/161178533)

### CODE REVIEW

File: server/controllers/categoryController.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
-import { Category, Article, ArticleCategory } from '../models';
+import {
+  Category,
+  Article,
+  ArticleCategory,
+  User
+} from '../models';
```
[the remark this answers is not in the document]
Comment (lauragift21) [reply server/controllers/categoryController.js:6]: Hound is flagging this as an error. which is fine since we don't add file extensions to file imports.

**Review by lauragift21 — APPROVED** [review]
/LGTM 👍

**Review by emekafredy — APPROVED** [review]
LGTM

**Review by iamuchejude — APPROVED** [review]
Good work

File: server/controllers/categoryController.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (CEOehis) [reply server/controllers/categoryController.js:6]: 💯

**Pull request merged**